src/current/_includes/common/telemetry-table.html

```diff
@@ -0,0 +1,932 @@
+<table>
+ <tr>
+  <th>Category</th>
+  <th>Description</th>
+  <th>Fields</th>
+  <th>Example</th>
+ </tr>
+ <tr>
+  <td>Node information</td>
+  <td>Information about the CockroachDB node reporting diagnostics.</td>
+  <td>
+   <ul>
+    <li>Node ID</li>
+    <li>Bytes</li>
+    <li>Key Count</li>
+    <li>Range Count</li>
+    <li>Uptime</li>
+   </ul>
+  </td>
+  <td>
+   <pre>
```

Comment:
I think this column is much more readable, especially at smaller viewing ratios that cause the lines to break, if you use `<code>` tags in every cell:
<img width="337" alt="image" src="https://github.com/user-attachments/assets/31a01176-82fa-4960-83e0-4db36843eea0">

versus

<img width="304" alt="image" src="https://github.com/user-attachments/assets/f45e21f5-70c4-4c43-b4ff-3157f58dce55">